### PR TITLE
Add page.codeberg.lazyt.ubpm

### PR DESCRIPTION
**Pull request by LazyT**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by LazyT** [comment]
bot, build page.codeberg.lazyt.ubpm

**Comment by LazyT** [comment]
bot, build page.codeberg.lazyt.ubpm

**Comment by hfiguiere** [comment]
I am having a hard time to see how a third build will pass when the first two didn't and there was no change.

File: page.codeberg.lazyt.ubpm.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - cp release/appbundle/lin/$FLATPAK_ID.appdata.xml $FLATPAK_DEST/share/metainfo
+      - cp sources/mainapp/hlp/*.q* $FLATPAK_DEST/bin/Guides
+      - cp sources/mainapp/lng/mainapp_*.qm sources/plugins/shared/plugin/lng/plugins_*.qm $FLATPAK_DEST/bin/Languages
+      - cp sources/plugins/*.so $FLATPAK_DEST/bin/Plugins
+      - cp sources/mainapp/qss/*.qss $FLATPAK_DEST/bin/Themes
+      - touch $FLATPAK_DEST/bin/.ubpm.flatpak
+    sources:
+      - type: git
+        url: https://codeberg.org/lazyt/ubpm.git
+        tag: '1.9.0'
```
Comment (hfiguiere) [inline page.codeberg.lazyt.ubpm.yml:28]: there should be a commit in addition to the tag.

File: page.codeberg.lazyt.ubpm.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+finish-args:
```
Comment (hfiguiere) [inline page.codeberg.lazyt.ubpm.yml:29]: finish-args are above module in general

File: page.codeberg.lazyt.ubpm.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+command: ubpm
+rename-desktop-file: ubpm.desktop
+rename-icon: ubpm
+modules:
+  - name: ubpm
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+    build-commands:
+      - cd sources && qmake && make
```
Comment (hfiguiere) [inline page.codeberg.lazyt.ubpm.yml:14]: buildsystem qmake should be used

File: page.codeberg.lazyt.ubpm.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: ubpm
+rename-desktop-file: ubpm.desktop
+rename-icon: ubpm
+modules:
+  - name: ubpm
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
```
Comment (hfiguiere) [inline page.codeberg.lazyt.ubpm.yml:12]: why this?

**Comment by LazyT** [comment]
> I am having a hard time to see how a third build will pass when the first two didn't and there was no change.

There was a commit: https://codeberg.org/LazyT/ubpm/commit/4cd4773456b4880b4921717afef89c62b5c13c68 to fix the manifest.

**Comment by hfiguiere** [comment]
retagging is bad. bad. bad.

File: page.codeberg.lazyt.ubpm.yml (hunk 1 above)
Comment (hfiguiere) [reply page.codeberg.lazyt.ubpm.yml:28]: this protects about retagging

**Comment by LazyT** [comment]
I never used Flatpak before, so first of all: sorry!

> finish-args are above module in general

Ok, will change this.

> buildsystem qmake should be used

If using "qmake" instead "simple" I get

`Error: module: ubpm: Can't find *.pro file`

How can I set "sources/ubpm.pro" to be used?

Thanks!

**Comment by hfiguiere** [comment]
```
       subdir (string)
           Build inside this subdirectory of the extracted sources

```

**Comment by LazyT** [comment]
Hopefully last question:

Can I set "no-make-install: true" and leave the rest as is or should I better use something like "install-rule: flatpak"?

Normally "make install" installs to "/opt/ubpm" and "/usr/share/<applications|icons>" on the system, which can't be used for flatpak. What to should I use as dirs in this case?

Thanks again!

**Comment by LazyT** [comment]
Forget the above questions, it was too late in the evening.

Please check the manifest again, I hope it's ok now?

Thank you!

**Comment by hfiguiere** [comment]
bot, build page.codeberg.lazyt.ubpm

File: page.codeberg.lazyt.ubpm.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+command: ubpm
+rename-desktop-file: ubpm.desktop
+rename-icon: ubpm
+finish-args:
+  - --socket=x11
+# wayland fails to open default browser?
+#  - --socket=wayland
+#  - --socket=fallback-x11
+  - --socket=cups
+  - --device=dri
```
Comment (hfiguiere) [inline page.codeberg.lazyt.ubpm.yml:14]: you probably need `--device=all` here. How does it work with USB?

Comment (LazyT) [reply page.codeberg.lazyt.ubpm.yml]: You are right. The USB plugins did not work, I changed it.

**Comment by LazyT** [comment]
bot, build page.codeberg.lazyt.ubpm

**Pull request closed without merging**